src/NuGet.Services.EndToEnd/Support/Package.cs

```diff
@@ -59,5 +60,38 @@ public static Package Create(PackageCreationContext context)
 
             return new Package(context.Id, context.NormalizedVersion, context.FullVersion, nupkgBytes);
         }
+
+        public static Package SignedPackage()
+        {
+            string id;
+            string normalizedVersion;
+            string fullVersion;
+
+            ReadOnlyCollection<byte> nupkgBytes;
+
+            using (var nupkgStream = File.Open(EnvironmentSettings.SignedPackagePath, FileMode.Open))
```

Comment:
if the path is not set this will throw. Check input and return null.

Reply:
Fixed. I made this method throw an exception that details that the environment variable must be set with a proper path as that'll be easier to debug.